azext_iot/central/_help.py

```diff
@@ -850,6 +1051,71 @@ def _load_central_device_groups_help():
             --app-id {appid}
     """
 
+    helps[
+        "iot central device-group show"
+    ] = """
+        type: command
+        short-summary: Get the device group by ID.
+
+        examples:
+        - name: Get the device group by ID
+          text: >
+            az iot central device-group show
+            --app-id {appid}
+            --device-group-id {devicegroupid}
+    """
+
+    helps[
+        "iot central device-group create"
+    ] = """
+        type: command
+        short-summary: Create or update a device group.
+
+        examples:
+        - name: Create or update a device group.
+          text: >
+            az iot central device-group create
+            --app-id {appid}
+            --device-group-id {devicegroupid}
+            --display-name {displayname}
+            --filter {filter}
+            --description {description}
+            --organizations {organizations}
+    """
+
+    helps[
+        "iot central device-group update"
+    ] = """
+        type: command
+        short-summary: Update a device group via patch.
+        long-summary: Update an existing device group by ID.
```

Comment:
for most of your one sentence long summaries, you can just replace the short summaries with the long ones. The long summaries are for multi line/specific instructions.